core/client/hdfs/src/main/java/alluxio/hadoop/AlluxioFileSystem.java

```diff
@@ -49,9 +48,4 @@
       throws IOException, URISyntaxException {
     super(uri, new FileSystem(), conf, Constants.SCHEME, false);
   }
-
-  @Override
-  public int getUriDefaultPort() {
-    return Integer.parseInt(PropertyKey.MASTER_RPC_PORT.getDefaultValue());
-  }
```

Comment:
Should this be getting removed?

Reply:
URL like `alluxio://test` will be treated as a logical name, so we should remove this statement.

Reply:
done